experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -469,6 +472,15 @@ function operators_lusztig(L::LieAlgebra, reduced_expression::Vector{Int})
   return operators
 end
 
+function demazurify_operators(V::DemazureModuleData, simple_operators::Vector{RootSpaceElem})
+  op = RootSpaceElem[]
+  inv_weylgroup_elem = inv(V.weyl_group_elem)
+  for root in simple_operators
+    push!(op, -(root * inv_weylgroup_elem))
+  end
+  return op
```

Comment:
```suggestion
  inv_weylgroup_elem = inv(V.weyl_group_elem)
  op = [-(root * inv_weylgroup_elem) for root in simple_operators]
  return op
```
or
```suggestion
  inv_weylgroup_elem = inv(V.weyl_group_elem)
  operators = map(simple_operators) do root
    -(root * inv_weylgroup_elem)
  end
  return operators
```
would be more efficient as julia can infer the final length of the vector (instead of increasing it multiple times with your push-loop)